obal/data/modules/check_copr_build.py

```diff
@@ -0,0 +1,54 @@
+"""
+Check if a build already exists in Copr
```

Comment:
scope creep: this could be a more generic `copr_package_info` module, where one of the results is whether it exists

but doesn't have to be in this PR, just food for thought